pootle/apps/virtualfolder/receivers.py

```diff
@@ -0,0 +1,13 @@
+
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from virtualfolder.models import VirtualFolder
+
+
+@receiver(post_save, sender=VirtualFolder)
+def vfolder_unit_priority_handler(sender, **kwargs):
+
+    for unit in kwargs["instance"].units.all():
```

Comment:
For the "virtualfolder location changed" scenario this should be moved to `VirtualFolder.save()` since in https://github.com/translate/pootle/blob/master/pootle/apps/virtualfolder/models.py#L144-L171 all relationships with units are unlinked and then relinked. There is the only place where you can recalculate priority for previously linked units that are no longer linked.